client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -0,0 +1,345 @@
+								</div>
+							</li>
+							<li>
+								<div className="domain-to-plan-nudge__header-features-item">
+									<Gridicon
+										className="domain-to-plan-nudge__header-features-item-checkmark"
+										icon="checkmark"
+										size={ 24 }
+									/>
+									{ translate( 'Upload up to 3GB of photos and documents' ) }
+								</div>
+							</li>
+							<li>
+								<div className="domain-to-plan-nudge__header-features-item">
+									<Gridicon
+										className="domain-to-plan-nudge__header-features-item-checkmark"
+										icon="checkmark"
+										size={ 24 }
+									/>
+									{ translate( 'Bundled with your domain for the best value!' ) }
+								</div>
+							</li>
+						</ul>
+					</div>
+				</div>
+				<div className="domain-to-plan-nudge__actions-group">
+					<div className="domain-to-plan-nudge__plan-price-group">
+						<div
+							className="domain-to-plan-nudge__discount-value">
+							{
+								translate( 'SAVE %(discount)s', {
+									args: {
+										discount: formatCurrency( rawDiscount, userCurrency )
+									}
+								} )
+							}
+						</div>
+						<PlanPrice
+							rawPrice={ rawPrice }
+							currencyCode={ userCurrency }
+							original
+						/>
+						<PlanPrice
+							rawPrice={ discountedRawPrice || rawPrice }
+							currencyCode={ userCurrency }
+							discounted
+						/>
+
+						<div className="domain-to-plan-nudge__plan-price-timeframe">
+							{ translate( 'for one year subscription' ) }
+						</div>
+					</div>
+					<div className="domain-to-plan-nudge__upgrade-group">
+						<Button
+							onClick={ this.oneClickUpgrade }
+							disabled={ ! storedCard || isSubmitting }
+							primary
+						>
+							{ isSubmitting
+								? translate( 'Completing your purchase' )
+								: translate( 'Upgrade Now for %s', {
+									args: formatCurrency( discountedRawPrice || rawPrice, userCurrency )
+								} )
+							}
+						</Button>
+						<div className="domain-to-plan-nudge__credit-card-info">
+							<a
+								className="domain-to-plan-nudge__credit-card-info-link"
+								href={ `/checkout/${ siteId }/personal` }>
+								{
+									translate( 'Using credit card ****%s', {
+										args: storedCard ? storedCard.card : 'xxxx'
+									} )
+								}
+							</a>
+						</div>
+					</div>
+				</div>
+			</DismissibleCard>
+		);
+	}
+}
+
+export default connect(
+	( state, props ) => {
+		const siteId = props.siteId || getSelectedSiteId( state ),
+			productSlug = PLAN_PERSONAL,
+			productId = plansList[ PLAN_PERSONAL ].getProductId();
+
+		return {
+			discountedRawPrice: getPlanDiscountedRawPrice( state, siteId, productSlug ),
+			hasFreePlan: isCurrentSitePlan(
+				state,
+				siteId,
+				plansList[ PLAN_FREE ].getProductId()
+			),
+			productId,
+			productSlug,
+			rawDiscount: getPlanRawDiscount( state, siteId, productSlug ),
+			rawPrice: getPlanRawPrice( state, productId ),
```

Comment:
Note to self: use `getSitePlanRawPrice` instead so we only need to depend on the site specific plans endpoint